src/pages/fr/reference/configuration-reference.md

```diff
@@ -0,0 +1,507 @@
+---
+# NOTE: This file is auto-generated from 'scripts/docgen.mjs'
+# Do not make edits to it directly, they will be overwritten.
+
+layout: ~/layouts/MainLayout.astro
+title: Référence de configuration
+i18nReady: true
+setup: |
+  import Since from '../../../components/Since.astro';
+---
+
+La référence suivante couvre toutes les options de configuration prises en charge dans Astro. Pour en savoir plus sur la configuration d'Astro, lisez notre guide sur la [configuration d'Astro](/fr/guides/configuring-astro/).
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config'
+
+export default defineConfig({
+  // Vos options de configuration ici...
+})
+```
+## Options de niveau supérieur
+
+### racine
+
+<p>
+
+**Type:** `string`<br>
+**CLI:** `--root`<br>
+**Par défaut:** `"."` (répertoire de travail actuel)
+</p>
+
+Vous ne devez fournir cette option que si vous exécutez la commande CLI `astro` dans un répertoire autre que le répertoire racine du projet. Habituellement, cette option est fournie via le CLI au lieu du [fichier de configuration Astro](/fr/guides/configuring-astro/#supported-config-file-types), car Astro doit connaître la racine de votre projet avant de pouvoir localiser votre fichier de configuration.
```

Comment:
The `#` part of the link here needs updating because that page is now in French:

```suggestion
Vous ne devez fournir cette option que si vous exécutez la commande CLI `astro` dans un répertoire autre que le répertoire racine du projet. Habituellement, cette option est fournie via le CLI au lieu du [fichier de configuration Astro](/fr/guides/configuring-astro/#types-de-fichier-de-configuration-supportés), car Astro doit connaître la racine de votre projet avant de pouvoir localiser votre fichier de configuration.
```